README.md

```diff
@@ -72,10 +72,6 @@ The rails-blog-app is a classic example of a blog website. It is a fully functio
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍